README.md

```diff
@@ -6,7 +6,7 @@
 
 # unistore
 
-> A tiny 650b centralized state container with component bindings for [Preact].
+> A tiny 666b centralized state container with component bindings for [Preact].
```

Comment:
Ends up being really hard to keep this in sync so I usually round to the nearest 50b mark.